multimodal/src/autogluon/multimodal/predictor.py

```diff
@@ -779,6 +804,7 @@ def _fit(
         hyperparameters: Optional[Union[str, Dict, List[str]]] = None,
         teacher_predictor: Union[str, MultiModalPredictor] = None,
         hpo_mode: bool = False,
+        init_only: bool = False,
```

Comment:
Do we need `init_only` for now?

Reply:
Removed. Let's work on to support evaluate before fit in next pr.